bot/cogs/last_fm.py

```diff
@@ -0,0 +1,206 @@
+import json
+import urllib
+import discord
+from bot.settings import MUSIC_TOKEN
+from discord.ext import commands
+
+
+class MusicCog(commands.Cog):
+    """
+    Commands for song searching
+    """
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(
+        name="getsong",
+        brief="gets info about a song",
+        description="takes in song by itself or song followed by artist seperated by ; ",
+        aliases=['song', 'gets']
+    )
+    async def getsong(self, ctx, *, args):
+        try:
+            if ', ' in args:
```

Comment:
I believe this `,` is supposed to be a `;`, please keep the splits consistent throughout the code.